src/PharoThings-Devices-HCSR04.package/PotHCSR04Device.class/README.md

```diff
@@ -0,0 +1,31 @@
+I'm a class to control the ultrasonic HCSR-04 sensors.
+You can use many ultrasonic sensors at the same time. Just change the PINs id. 
+
+To use:
+(inspector)
+ultrasonic := board installDevice: PotHCSR04Device new.
+
+(playground. change the board model  to your board)
+ultrasonic := (RpiBoard3B current) installDevice: PotHCSR04Device new.
+
+You can name the object using
+name: 'Left sensor'
+
+To read the distance use one of the method below. 
+
+readDistance. "It will return a number"
+printDistance. "It will return a string"
+
+I'm using the GPIO0 to triggerPin and GPIO21 to echoPin as default configuration.
+If you want to choose your own PIN position, you can set using the PIN Id (BCM number):
+
+triggerPin: 27 echoPin: 17.
+
+Some of my sensors brothers uses only 1 GPIO to send and read the ultrasonic pulse. 
```

Comment:
But it still same model?

Reply:
There are different models (HC-SR04, SRF05, SRF06, DYP-ME007, JSN-SR04T & Parallax PING) but they work in the same way. The 3 pins models (PWR,GND,SGN), they use the same PIN to send the pulse and read it. The 4 pins models (PWR,GND,TRG,ECH) they use 1 PIN to send the ultrasonic pulse and other PIN to read the time of duration the wave spend to go and back. 
This code support the 2 versions, using 3 and 4 PINs.